src/lexer.mll

```diff
@@ -68,6 +70,7 @@ if the string length of Buffer is 0, this lexeme is the first char following dot
 { if Buffer.contents buf |> String.length > 0 then
 (Buffer.add_string buf (Lexing.lexeme lexbuf); read_class buf lexbuf)
 else read lexbuf }
+| single_quotation { Buffer.add_string buf "'"; read_class buf lexbuf }
```

Comment:
Is it compiled successfully? Maybe `tailwindcss_single_quotation`?

Reply:
Fixed!